android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -81,6 +80,55 @@
     private static final String SDK_TAG = "AppLovinSdk";
     private static final String TAG     = "AppLovinMAXModule";
 
+    private static final String ON_BANNER_AD_LOADED_EVENT     = "OnBannerAdLoadedEvent";
```

Comment:
Let me see if I can improve the structure of this code.

Reply:
I was going to update to:

```
    private static class Callback
    {
        private static class BANNER
        {
            private static final String ON_AD_LOADED_EVENT      = "OnBannerAdLoadedEvent";
            private static final String ON_AD_LOAD_FAILED_EVENT = "OnBannerAdLoadFailedEvent";
            private static final String ON_AD_CLICKED_EVENT     = "OnBannerAdClickedEvent";
            private static final String ON_AD_COLLAPSED_EVENT   = "OnBannerAdCollapsedEvent";
            private static final String ON_AD_EXPANDED_EVENT    = "OnBannerAdExpandedEvent";
            private static final String ON_AD_REVENUE_PAID      = "OnBannerAdRevenuePaid";
        }

        private static class MREC
        {
            private static final String ON_LOADED_EVENT      = "OnMRecAdLoadedEvent";
            private static final String ON_LOAD_FAILED_EVENT = "OnMRecAdLoadFailedEvent";
            private static final String ON_CLICKED_EVENT     = "OnMRecAdClickedEvent";
            private static final String ON_COLLAPSED_EVENT   = "OnMRecAdCollapsedEvent";
            private static final String ON_EXPANDED_EVENT    = "OnMRecAdExpandedEvent";
            private static final String ON_REVENUE_PAID      = "OnMRecAdRevenuePaid";
        }
 ...
    }
```

But I think it's about the same.

Reply:
Can we auto-format the file though?

Reply:
Yes, I did run the format and optimize-imports.

Reply:
Auto-formatted it for you here: https://github.com/AppLovin/AppLovin-MAX-React-Native/commit/2d8a25d684760131d685366929a59b3a67287916

Reply:
Thanks!